youtube_dl/extractor/intldropout.py

```diff
@@ -0,0 +1,152 @@
+                'upload_date': '20181206',
+                'timestamp': 1544117975,
+            }
+        },
+        {
+            'url': 'https://intl.dropout.tv/videos/um-actually-behind-the-scenes',
+            'md5': 'b974927cd563423fe50945dbfdbb894c',
+            'info_dict': {
+                'id': '397943',
+                'ext': 'mp4',
+                'title': 'Um, Actually: Behind the Scenes',
+                'thumbnail': r're:^https://vhx.imgix.net/.*\.jpg$',
+                'description': 'What does it take to stump the nerdy? Mike Trapp and team pull back the curtain.',
+                'upload_date': '20181206',
+                'timestamp': 1544118409,
+            }
+        }
+    ]
+
+    def _real_initialize(self):
+        self._login()
+
+    def _login(self):
+        email, password = self._get_login_info()
+        if email is None or password is None:
+            if self._downloader.params.get('cookiefile') is None:
+                raise ExtractorError('No login info available, needed for using %s.' % self.IE_NAME, expected=True)
+            return True
+
+        login_page = self._download_webpage(
+            self._LOGIN_URL, None,
+            note='Downloading login page',
+            errnote='unable to fetch login page', fatal=False,
+            expected_status=200
+        )
+
+        if login_page is False:
+            return
+
+        login_form = self._hidden_inputs(login_page)
+
+        login_form.update({
+            'passwordless': 0,
+            'email': email,
+            'password': password
+        })
+
+        request = sanitized_Request(
+            self._LOGIN_URL, urlencode_postdata(login_form))
+        request.add_header('Content-Type', 'application/x-www-form-urlencoded')
+        try:
+            self._download_webpage(request, None, 'Logging in', expected_status=302)
+        except Exception:
+            raise ExtractorError(
+                'Unable to login',
+                expected=True)
+
+    def _real_extract(self, url):
+        try:
+            webpage = self._download_webpage(url, None, expected_status=200)
+        except Exception:
+            raise ExtractorError(
+                'Unable to fetch page',
+                expected=True)
+        video = self._html_search_regex(r'<iframe[^>]*"(?P<embed>https://embed.vhx.tv/videos/[0-9]+[^"]*)"[^>]*>', webpage, 'embed')
+        video_id = self._search_regex(r'https://embed.vhx.tv/videos/(?P<id>[0-9]+)', video, 'id')
+        video_title = self._html_search_regex(r'<h1 class="[^"]*video-title[^"]*"[^>]*>(<strong>)?(?P<title>[^<]+)<', webpage, 'title')
+        return self.url_result(video, video_id=video_id, video_title=video_title)
+
+
+class IntlDropoutPlaylistIE(IntlDropoutIE):
+    IE_NAME = 'intldropout:playlist'
+    _VALID_URL = r'^https://intl\.dropout\.tv/(?P<id>[^/]+(/season:[^/]+)?)$'
```

Comment:
No trailing $.

Reply:
without the trailing $, some video Links would match as Playlist

Reply:
Override `suitable`.

Reply:
thanks. fixed it.